include/openenclave/bits/result.h

```diff
@@ -344,6 +344,12 @@ typedef enum _oe_result
      */
     OE_ALREADY_INITIALIZED,
 
+    /**
+     * The hash of claims section in the quote does not match the hash value
+     * held in the report_data field. This occurs when claims are tampered.
+     */
+    QE_QUOTE_HASH_MISMATCH,
+
```

Comment:
I am curious why it's QE_QUOTE_HASH_MISMATCH, not OE_QUOTE_HASH_MISMATCH.
Any special reason?

Reply:
Right, it should be OE_

---
In reply to: [397695413](https://github.com/openenclave/openenclave/pull/2706#discussion_r397695413) [](ancestors = 397695413)

Reply:
Yes...It should be OE. Sorry for the typo.